src/nodes/errors.ts

```diff
@@ -45,9 +45,9 @@ class ErrorNodeConnectionInfoNotExist extends ErrorNodes {}
 
 class ErrorNodeConnectionPublicKeyNotFound extends ErrorNodes {}
 
-class ErrorInvalidNodeId extends ErrorNodes {
-  description: string = 'Invalid node ID.';
-  exitCode: number = 64;
+class ErrorNodeIdInvalid extends ErrorNodes {
+  description = 'Invalid node ID';
```

Comment:
@tegefaulkes all descriptions should not use the types `: string`. This causes problems later.

Also remember that `.` should not be used here. Ensure your comments and messages are consistent.

Reply:
This wasn't changed in #318 from what I recall. and updating the errors was in the scope of #310.

Reply:
This is being removed here and replaced with `throw new validationErrors.ErrorParse` now.

Reply:
And acknowleged on the #310 updating `nodes/errors.ts` with `description` and `exitCode`.